src/devices/Charlieplex/tests/tests.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
NIT: Usually we call the UnitTests projects like:
<BindingName>.Tests.csproj so that they run as expected and test logs are identifyiable to a binding so please rename this to be Charlieplex.Tests.csproj